src/components/dashboard/Dashboard.test.jsx

```diff
@@ -49,14 +49,14 @@ jest.mock('react-router-dom', () => ({
   useLocation: () => (mockLocation),
 }));
 
-jest.mock('./sidebar/offers', () => ({
-  ...jest.requireActual('./sidebar/offers'),
-  fetchOffers: () => ({ type: 'fetch' }),
-}));
-
 describe('<Dashboard />', () => {
+  const defaultOffersState = {
+    offers: [],
+    loading: false,
+    offersCount: 0,
```

Comment:
random thought: do we need to have `offersCount` as a separate key in the offers state when we could just as easily read `offers.length` where relevant?

Reply:
We got offersCount from the API for free, so, might as well use it was my thought. ;)

Reply:
Just recapping from our conversation in the Slack thread, the `offers` and `offersCount` is coming from the reducer, not the API response.

Reply:
Yeah, ticket is opened to deal with that issue.